.github/workflows/gradle.yml

```diff
@@ -1,75 +1,77 @@
-        path: ./tmp/rca
-    - name: Checkout Performance Analyzer package
-      uses: actions/checkout@v2
-      with:
-        path: ./tmp/pa
+      - name: Set up JDK ${{ matrix.java }}
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java }}
+
+      - name: Checkout Performance Analyzer
+        uses: actions/checkout@v2
+
+      - name: Checkout OpenSearch
+        uses: actions/checkout@v2
+        with:
+          repository: 'opensearch-project/OpenSearch'
+          path: OpenSearch
+          ref: '1.0.0-beta1'
+
+      - name: Build OpenSearch
+        working-directory: ./OpenSearch
+        run: ./gradlew publishToMavenLocal -Dbuild.version_qualifier=beta1 -Dbuild.snapshot=false -x test --info
+
+      - name: Checkout RCA
+        uses: actions/checkout@v2
+        with:
+          repository: 'opensearch-project/performance-analyzer-rca'
+          path: performance-analyzer-rca
+
+      - name: Build RCA
+        working-directory: ./performance-analyzer-rca
+        run: ./gradlew publishToMavenLocal -x test --info
+
       # fetch the main branch to make it available for spotless ratcheting
-    - name: Fetch 'main' branch
-      working-directory:  ./tmp/pa
-      run: git fetch --depth=1 origin main
-    - name: Set up JDK 1.12
-      uses: actions/setup-java@v1
-      with:
-        java-version: 1.12
-    - name: Build RCA with Gradle
-      working-directory:  ./tmp/rca
-      run: ./gradlew build -x test
-    - name: Publish RCA jar to maven local
-      working-directory: ./tmp/rca
-      run: ./gradlew publishToMavenLocal
-    # dependencies: OpenSearch
-    - name: Checkout OpenSearch
-      uses: actions/checkout@v2
-      with:
-        repository: 'opensearch-project/OpenSearch'
-        path: OpenSearch
-        ref: '1.0.0-beta1'
-    - name: Build OpenSearch
-      working-directory: ./OpenSearch
-      run: ./gradlew publishToMavenLocal -Dbuild.version_qualifier=beta1 -Dbuild.snapshot=false
-    - name: Build PA gradle using the new RCA jar
-      working-directory: ./tmp/pa
-      run: rm -f licenses/performanceanalyzer-rca-1.0.0.0-beta1.jar.sha1
-    - name: Update SHA
-      working-directory: ./tmp/pa
-      run: ./gradlew updateShas
+      - name: Fetch Performance Analyzer 'main' branch
+        run: git fetch --depth=1 origin main
+
+      - name: Update Performance Analyzer SHA for current build
+        run: |
+          rm -f licenses/performanceanalyzer-rca-1.0.0.0-beta1.jar.sha1
+          ./gradlew updateShas --info
+
       # Explicitly set the docker-compose program path so that our build scripts in RCA can run the program
       # This is necessary because of the Github Actions environment and the workingDir of the Gradle environment
-    - name: Set docker-compose path
-      run: DOCKER_COMPOSE_LOCATION=$(which docker-compose)
+      - name: Set docker-compose path
+        run: DOCKER_COMPOSE_LOCATION=$(which docker-compose)
```

Comment:
This had some issue. I fixed this recently in this commit.

https://github.com/opensearch-project/performance-analyzer-rca/pull/9/commits/171b79b419a801cbaa41f5e52b1b40a6b1f1e020